### DISCUSSION TITLE
为自主权作准备：企业使用开源软件的正确姿态 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#55] [Announcements]
# 为自主权作准备：企业使用开源软件的正确姿态 - 「开源之道」

https://opensourceway.community/posts/opensource_enterprise_guide/how-to-use-oss-in-companies/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

企业的自主权, 这个切入点, 可能和开源的社会效益相比无法融洽;
因为, 如果企业自主权高于一切的话,
那么无论开源世界多美好, 
一切都从头自主开发, 是自主权最高的策略;

因为, 毕竟, 软件市场并不是 100% 开源模式,
主要项目都是闭源项目,
那么, 自主并快速, 交付足够功能,
比高速交付市场上最好的功能, 
要来的更加有利,

企业拥抱开源, 还是得说提高效能/利润来分析?
